app/viam_client.go

```diff
@@ -63,6 +64,16 @@ func CreateViamClientWithAPIKey(
 	return CreateViamClientWithOptions(ctx, options, logger)
 }
 
+// Billingclient initializes and returns a Billingclient instance used to make app method calls.
+// To use Billingclient, you must first instantiate a ViamClient.
+func (c *ViamClient) Billingclient() *BillingClient {
```

Comment:
```suggestion
func (c *ViamClient) BillingClient() *BillingClient {
```
I think you can capitalize the `c` here without a namespacing issue (see `DataClient` method below.)